docs/remote-snapshotter.md

```diff
@@ -0,0 +1,65 @@
+This document is about the architecture used to support remote snapshotters in firecracker-containerd. If you would like to jump in and launch estargz images on firecracker-containerd, see the [remote snapshotter getting started guide.](remote-snapshotter-getting-started.md)
+
+# Remote snapshotters overview
+
+Remote snapshotters are a type of [containerd snapshotter](https://github.com/containerd/containerd/blob/ff91434af1b937a31cd3f76c31d079b74a11d9d0/docs/snapshotters/README.md) that are responsible for fetching snapshots from a remote location. When pulling an image with a normal snapshotter, the snapshotter provides mount points, but the containerd client pulls image layers into the containerd content store and unpacks them onto those mount points. With remote snapshotters, the snapshotter is responsible for providing mount points as well as pulling image layers and populating that snapshot. This allows remote snapshotters such as the [stargz-snapshotter](https://github.com/containerd/stargz-snapshotter) to lazily load snapshot content on-demand, thus reducing the upfront time needed for image pull. For more details see [containerd's remote snapshotter documentation](https://github.com/containerd/containerd/blob/v1.6.6/docs/remote-snapshotter.md).
```

Comment:
How about using one of the release tags instead of raw Git commit hashes?

Reply:
I tried to where I could, but the snapshotters readme was created after the 1.6 tag and won't be backported, so it's not in any tag until 1.7.

Reply:
I didn't know that. Thanks!